homeassistant/components/homekit_controller/manifest.json

```diff
@@ -4,7 +4,7 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/homekit_controller",
   "requirements": [
-    "aiohomekit==0.2.54"
+    "aiohomekit==0.2.57"
```

Comment:
Don't forget to mark the dependency upgrade in the MR template.

Reply:
Of course happy to do this, but the PR template says to only check 1 box.